runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingSpotOverlay.java

```diff
@@ -42,6 +44,9 @@
 	private final FishingPlugin plugin;
 	private final FishingConfig config;
 
+	@Inject
+	private Client client;
```

Comment:
Inject this in constructor (and move FishingConfig injection there too and make the FishingSpotOverlay ctor private)

Reply:
I moved the line 48 into the ctor and made the ctor private, not sure what you mean by the FishingConfig injection?

Reply:
Oh sorry, I meant ItemManager

Reply:
I removed client because it wasn't actually used in the code and I forgot to remove it before, but if I move ItemManager into ctor then itemManager in getFishImage() becomes unresolved so not sure what to do here

Reply:
Assign it to private final variable in constructor